testing/capybara-all-does-not-return-array.md

```diff
@@ -0,0 +1,40 @@
+# Capybara `all` does not return an array
+
+Although it looks like it returns an `Array`, Capybara's `all` finder actually
+returns a `Capybara::Result`.  Turns out it returns a `Capybara::Result` object.
```

Comment:
Good catch! I refactored that opening paragraph and forgot to remove this sentence.